src/cmd/linuxkit/build.go

```diff
@@ -49,6 +49,7 @@ func build(args []string) {
 	buildSize := buildCmd.String("size", "1024M", "Size for output image, if supported and fixed size")
 	buildPull := buildCmd.Bool("pull", false, "Always pull images")
 	buildDisableTrust := buildCmd.Bool("disable-content-trust", false, "Skip image trust verification specified in trust section of config (default false)")
+	buildVMLinux := buildCmd.Bool("vmlinux", false, "Decompress the Linux kernel (default false)")
```

Comment:
The name of this option is a bit opaque for what it does. I think it might be implying "vmlinux not vmlinuz" or some such, but that's a bit of an obscure internal detail for an end user.

`-decompress-kernel` perhaps?

Same sort of thing should be used for the variable names too I reckon.